### PR TITLE
fix: handle multiple affiliations properly

### PR DESCRIPTION
**Pull request by fernandolguevara**
context #1633

### CODE REVIEW

**Comment by gabestein** [comment]
Thanks so much for this quick fix @fernandolguevara! Looks great, there was a check for affiliation length I added so it wouldn't print a 0 by names with no affiliations. Adding one of our engineers now to take a final look.

If you're looking to make contributions in the future, please feel free to reach out to hello@pubpub.org or comment on an issue you'd like to take on. We don't have great contribution guidelines, but it's something we'd like to support more of. In general, we like to make sure PRs include:

- a high-level description of what the PR accomplishes
- screenshots/recordings of the differences if applicable/possible
- a test plan for other devs to follow

See, e.g.:  https://github.com/pubpub/pubpub/pull/2041

**Comment by fernandolguevara** [comment]
@gabestein 

Can u change the Author 2 to be affiliated to `Knowledge Futures; Wassamatu U`?  and regenerate the PDF, please?

**Comment by gabestein** [comment]
Ah, I see. Sorry I didn't test that. We'll have to have a chat about semicolons vs. commas as the separator. Ultimately, we plan to tokenize the field to support multiple institutions, so we'll want something that's fairly distinct to do a migration.

File: workers/tasks/export/html.tsx [hunk 1]
```diff
@@ -206,17 +221,20 @@ const renderFrontMatter = ({
...
-								affiliation && affiliations.includes(affiliation)
-									? 1 + affiliations.indexOf(affiliation)
-									: null;
+							const affs = getAffiliations(attr);
 							return (
 								<span className="name" key={index}>
 									{fullName}
-									{affiliationNumber && <sup>{affiliationNumber}</sup>}
-									{index < attributions.length - 1 && ', '}
+									{affs?.length > 0 &&
```
Comment (idreyn) [inline workers/tasks/export/html.tsx:229]: You could skip this check since if `affs.length === 0` the `map()` will just not render anything.

**Review by idreyn — APPROVED** [review]
Looks good to me, thanks for this change!

**Pull request merged**